src/Data/Cursor/AbstractCursor.php

```diff
@@ -132,6 +144,153 @@ protected function is_valid_offset_and_node() {
 		return true;
 	}
 
+	/**
+	 * Validates threshold field configuration. Validation failure results in a fatal
```

Comment:
This is an example of my [earlier feedback](https://github.com/wp-graphql/wp-graphql/pull/2882#pullrequestreview-1578197213). Ideally this doc-block would say _what_ a valid threshold field configuration looks like.